test/test-v8debugapi.ts

```diff
@@ -26,12 +16,13 @@
 
 var breakpointInFoo = {
   id: 'fake-id-123',
-  location: { path: 'test-v8debugapi.js', line: 4 }
+  // TODO: Determine if we should be restricting to only the build directory.
+  location: { path: 'build/test/test-v8debugapi-code.js', line: 4 }
```

Comment:
I think this is because this test has a slightly different `workingDirectory` option set compared to the other tests.